app/next-client-app/app/datasets/page.tsx

```diff
@@ -19,6 +19,7 @@ interface DataSetListProps {
 }
 
 export default async function DataSets({ searchParams }: DataSetListProps) {
+  // TODO: ADD default page size here
```

Comment:
How come?

Reply:
Yeah, sorry. The default page size, already defaulted by 10,  is not needed here. I will remove this TODO in the next PR. Thanks